tcms/templates/plan/get_cases.html

```diff
@@ -215,27 +186,3 @@
 	</span>
 	<span class="js-nomore-msg" style="display:none">All cases have been loaded.</span>
 </div>
-{% verbatim %}
-	<script id="batch_tag_form_template" type="text/x-handlebars-template">
-		<form id="id_batch_tag_form">
-			<div class="dia_title" style=" margin:30px 20px;">Please type tag name:</div>
-			<div class="dia_content" style=" margin:30px 20px;">
```

Comment:
dia_title and dia_content classes are not used anywhere else. Remove from CSS.

Reply:
Done